data/www-widgetapi.js

```diff
@@ -0,0 +1,159 @@
+(function(){var g,k=this;function l(a){a=a.split(".");for(var b=k,c;c=a.shift();)if(null!=b[c])b=b[c];else return null;return b}
```

Comment:
We may want to add their copyright header or URL+version in the comment header or something
Otherwise in a week or three it's just a random minimized JS file in our repo that we don't know where it came from.